### ISSUE TITLE
when I specify asix=1 in cross_entropy of neural_network_raw it does not work

### ISSUE THREAD

**Issue by grvmeratwal** [aymericdamien/TensorFlow-Examples#368]
As you have used axis=1 in  cross_entropy of logistic_regression.ipynb

If I do same thing in neural_network_raw , it does not work
it gives accuracy of 12%

return tf.reduce_mean(-tf.reduce_sum(y_true * tf.math.log(y_pred),axis=1))

**Comment by grvmeratwal** [comment]
return tf.reduce_mean(-tf.reduce_sum(y_true * tf.math.log(y_pred),axis=1))
does not works in neural_network_raw

**Comment by grvmeratwal** [comment]
when I make learning_rate=0.05
it works with axis=1
